obsproject/obs-studio
# decklink-ouput-ui: Fix memory leak

### Description



Add missing `bfree` by changing `char *` to `BPtr<char>` in `DecklinkOutputUI::SavePreviewSettings()`.

### Motivation and Context




I don't want to have memory leaks.

### How Has This Been Tested?




1. Start OBS Studio
2. Open Decklink Output dialog
3. Close dialog
4. Exit OBS Studio and check there is no memory leaks.

### Types of changes

- Bug fix (non-breaking change which fixes an issue)







### Checklist:
